### ISSUE TITLE
[wg/wot] Web of Things Working Group rechartering

### ISSUE THREAD

**Issue by ashimura** [w3c/strategy#509]
**Note:** This is a placeholder issue for the WoT WG's rechartering process. The [WoT WG Charter](https://www.w3.org/2023/10/wot-wg-2023.html) is expiring on 2 October 2025.

New charter proposal, reviewers please take note.

# Charter Review

[Draft Charter (WIP)](https://w3c.github.io/wot/charters/wot-wg-2025-draft.html)

[diff from charter template](https://services.w3.org/htmldiff?doc1=https://w3c.github.io/charter-drafts/charter-template.html&doc2=https://w3c.github.io/wot/charters/wot-wg-2025-draft.html)

Expected end of charter refinement phase: 2026-03-31

If applicable:

[diff from previous charter](https://services.w3.org/htmldiff?doc1=https://www.w3.org/2023/10/wot-wg-2023.html&doc2=https://w3c.github.io/wot/charters/wot-wg-2025-draft.html)

[chair dashboard](https://www.w3.org/PM/Groups/chairboard.html?gid=wg/wot)

What kind of charter is this? Check the relevant box / remove irrelevant branches. 

* Existing 
 - [x] Existing WG recharter 
 - [ ] Existing IG recharter
 - If this is a charter extension or revision, any issue discussion:

Horizontal Reviews: apply the Github label "Horizontal review requested" to request reviews for accessibility (a11y), internationalization (i18n), privacy, security, and TAG. **Also add a "card" for this issue to the [Strategy Funnel](https://github.com/orgs/w3c/projects/97/views/2).**

## Known or potential areas of concern

Where would charter proponents like to see issues raised? (this strategy funnel issue, a different github repo, email, ...)

## Anything else we should think about as we review? 

Note: proposed chairs should be copied @... on this issue.

**Comment by koalie** [comment]
Charter refinement initiated: https://lists.w3.org/Archives/Public/public-wot-wg/2026Feb/0011.html

**Comment by dontcallmedom** [comment]
* the tentative deliverable is named "WoT Thing Protocol" in the charter, but the spec uses "Web Thing Protocol" as a title (which sounds less redundant than Web of Things Thing Protocol :)
* the list of groups for coordination needs to be revisited - at two groups have closed (Web &Networks, Efficient Extensible Interchange), one has been renamed (Privacy IG → Privacy WG),  and there is at least one duplicate (autonomous agent)
* it's not clear why the Bindings are to be published as informative documents, esp. given that their editors drafts come with conformance sections indicating they contain normative text
* shouldn't "Web of Things in a Nutshell" link to https://www.w3.org/WoT/documentation/ rather than https://www.w3.org/WoT/ ?
* the charter doesn't mention other Rec-track documents from the existing WG (architecture, discovery, profiles) - is that on purpose?

**Comment by himorin** [comment]
No comment or request from i18n

**Comment by sebastiankb** [comment]
Note: This is a draft response; it needs group approval on Wednesday.

> the tentative deliverable is named "WoT Thing Protocol" in the charter, but the spec uses "Web Thing Protocol" as a title (which sounds less redundant than Web of Things Thing Protocol :)

All the deliverables so far have the prefix "WoT" at the beginning. Including the "Web Thing Protocol" deliverable we decided to be consistent with all other WoT deliverables and decided to rename it to "WoT Thing Protocol". This should avoid confusing the users about the WoT technology. 

> the list of groups for coordination needs to be revisited - at two groups have closed (Web &Networks, Efficient Extensible Interchange), one has been renamed (Privacy IG → Privacy WG), and there is at least one duplicate (autonomous agent)

Thank you for the hint. There is already a PR working on this subject:

https://github.com/w3c/wot/pull/1286

> it's not clear why the Bindings are to be published as informative documents, esp. given that their editors drafts come with conformance sections indicating they contain normative text

Here are some points as response:

- protocol bindings are published as Notes because the describe how existing protocols are used in combination with the WoT Thing Description, rather than defining a new normative specifications which would expect a close cooperation with the SDO that develops the protocol 
- some protocols evolve rapidly. Publishing bindings as Notes allows faster updates and maintenance without the overhead of the REC track
- publishing bindings as Notes keeps the option open to later upgrade the deliverable into W3C REC

> shouldn't "Web of Things in a Nutshell" link to https://www.w3.org/WoT/documentation/ rather than https://www.w3.org/WoT/ ?

Thanks for the hint. Here is already a PR available: https://github.com/w3c/wot/pull/1287

> the charter doesn't mention other Rec-track documents from the existing WG (architecture, discovery, profiles) - is that on purpose?

The charter mention in the Scope section "Update and Refactor Existing Deliverables" which implicit covers all existing REC documents. However, we can mention all existing RECs again in the new charter if needed.

**Comment by dontcallmedom** [comment]
thanks, this addresses my comments except for the bit around Notes for bindings- the question of Rec-track vs Note-track should not be about pace of evolution (e.g. a Rec-track deliverable can iterate at CR while the protocol evolves), and upgrading a Note to a Rec is problematic from an IPR perspective. The editors drafts themselves are a bit unclear about whether they're meant to be informative guidance (which the abstract suggests and would be what a Note is meant to be) or a specification for interoperable behaviour (which the existence of a conformance section suggests, and would suggests making them Rec-track).

I could image either option to serve the community, but the group should be clear about its objectives and expectations with bindings.

**Comment by sebastiankb** [comment]
Thank you very much for your response. We will discuss this matter at today’s main meeting and see what the group thinks. I'm currently going through some existing W3C Notes, and many of them are also have normative aspects in there, such as https://www.w3.org/TR/wpub/ or https://www.w3.org/TR/did-extensions/

 I think the boundaries are sometimes blurred, and both options can be reasonable.

**Comment by dontcallmedom** [comment]
The publication of WPUB as a note was what was expected to stop normative work in what the Process document suggested in 2019.
The "conformance" of DID Extensions is about the registration process, not an how DID products interoperate; I'm unclear why they did this in a Note rather than a Registry track document, but I don't think it illustrates a case similar to the WoT bindings.

**Comment by ruoxiran** [comment]
no comment or request from APA.

**Comment by sebastiankb** [comment]
Sorry for the late response. In the last main call on Wednesday, we agreed on the resolution that all bindings should go with the Note track:

https://pad.w3.org/p/2026-05-13-wot

We are currently working on different PRs that will remove all the Conformance Sections of the existing editor drafts. Also see https://github.com/w3c/wot-binding-templates/issues/463

**Comment by jyasskin** [comment]
Thanks for sending the TAG this review. We have no architectural concerns about most of this work, but we noticed one issue. The Web of Things allows Things to be accessible over the public internet. This makes proper authentication and authorization a requirement (e.g. in [discovery](https://www.w3.org/TR/wot-architecture11/#sec-discovery-introductions)). However, the charter does not say that the group will coordinate with any authentication-related standards bodies, such as OpenID or the FedCM WG. It should.

**Comment by sebastiankb** [comment]
@jyasskin thanks for your feedback. The WoT deliverables, such as the WoT Thing Description, cover [security considerations](https://w3c.github.io/wot-thing-description/#sec-security-consideration). There is also a general [WoT Security and Privacy Guidelines](https://www.w3.org/TR/wot-security/) document available. However, you are right that the Charter is somewhat silent on existing security and privacy considerations, which will continue to be maintained. In this week’s WoT main call, we agreed to update this information accordingly:
https://github.com/w3c/wot/pull/1298

**Comment by simoneonofri** [comment]
From the security side, we have only one point (related TAG comment)

We would like to ask is whether it would be possible to make the **Security and Privacy Guidelines** a required deliverable, to be published as a W3C Note.

In particular, since the document describes the threat model and the related security and privacy risks, it would be useful, if possible, to also have cross-links from those threats to the places in the relevant WoT specifications where the corresponding mitigations or remediations are defined.

I think this would make the security story easier to follow, both for horizontal review and also for implementers.

**Comment by npdoty** [comment]
Privacy review had similar comment -- previously the group had worked on a security and privacy guidelines document, but this charter appears to drop it. That seems like something that would be useful to update as the group takes on new deliverables. And it seems especially significant because the problems in the marketplace with privacy/security around internet of things have definitely not been addressed; it's a constant threat and annoyance.

I think picking up that document again and maintaining it would be useful, but if the group has other ideas or other updates on the status, it would be useful for us to know.

**Comment by sebastiankb** [comment]
in today's WoT main call we agreed to merge the PR https://github.com/w3c/wot/pull/1298 which includes the requested security and privacy aspects.

**Comment by plehegar** [comment]
[Proposed W3C Charter: Web of Things (WoT) Working Group (until 2026-08-31)](https://lists.w3.org/Archives/Public/public-new-work/2026Jul/0003.html)

**Comment by simoneonofri** [comment]
Just adding the label for the review ([per our comment](https://github.com/w3c/strategy/issues/509#issuecomment-4591296709)) and thanks for https://github.com/w3c/wot/pull/1298

**Comment by plehegar** [comment]
https://lists.w3.org/Archives/Public/public-new-work/2026Sep/0004.html
